pages/ksphere/konvoy/latest/download/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: layout.pug
+navigationTitle: Download
+title: Download Konvoy
+menuWeight: 2
+excerpt: Learn how to download an instance of Konvoy 
+enterprise: false
+---
+
+When you are ready to download an instance of Konvoy, there are 2 possible scenarios:
+
+1. You have already spoken to your sales representative and requested a download link. See <a href="#Downloading_Email">Downloading Konvoy via Email Link</a> for intructions. 
+2. You want to try a trial version of Konvoy and are interested in downloading a trial instance of the product. See  <a href="#Downloading_Email">Downloading Konvoy via the Trial Page</a> for instructions. 
+
+## <a name="Downloading_Email">Downloading Konvoy via Email Link</a>
+
+After you contact your sales representative, an automated email from our license server will be sent to your account's primary contact with instructions on how to download Konvoy. Select the correct link within the email to start your download. 
```

Comment:
Replace the word "account" with "company".  I think account is ambiguous which could mean a login account or something - as opposed to the primary contact at their company - the person that acquired the license.
 
Might be better to have:
Licensed customers should have received an automated e-mail from the D2iQ entitlement server upon completion of a purchase.  The e-mail is sent to your company's primary support contact with instructions on how to download Konvoy.  If you have questions please contact your sales representative or send an e-mail to sales@d2iq.com.